circleguard/clock.py

```diff
@@ -7,25 +7,27 @@ def __init__(self):
         self.current_speed = 1
         self.paused = False
         self.paused_at_run_time = None
-        self.last_run_time = time.time_ns()
+        self.last_run_time = time.perf_counter()
```

Comment:
Did you change to `perf_counter` over `time_ns` for finer granularity, or another reason? The former seems silly - nanosecond is plenty accurate. 

I ask because there are caveats that come with using a timer with an undefined measurement unit. From [the docs](https://docs.python.org/3/library/time.html#time.perf_counter): `Return the value (in fractional seconds) of a performance counter, i.e. a clock with the highest available resolution to measure a short duration`. That "highest available resolution" may differ between systems. https://stackoverflow.com/a/25787875/12164878 defines it as a "unitless measurement", and from the docs and what I just said I think he's entirely right to call it that, but I believe you are "converting" it to milliseconds and treating it as milliseconds, not unitless.

Reply:
Initially this change was added to support over 1k fps.